docs/components/modeler/desktop-modeler/model-your-first-diagram.md

```diff
@@ -1,28 +1,35 @@
 ---
 id: model-your-first-diagram
 title: Model your first diagram
+description: "Learn how to model your first diagram using Desktop Modeler and BPMN."
 ---
 
 After starting [Desktop Modeler](./index.md), you can model your first BPMN diagram. Follow the steps below:
 
-1. Create a [BPMN](../bpmn/bpmn.md) diagram:
+1. Create a [BPMN](../bpmn/bpmn.md) diagram by selecting **BPMN diagram**:
 
 ![empty application](./img/empty.png)
 
-2. View the BPMN diagram with a start event:
+The BPMN diagram opens with a start event. [Events](/components/modeler/bpmn/events.md) in BPMN represent things that happen. A process can react to events and emit events, for example:
 
 ![new diagram](./img/new-diagram.png)
 
-3. On the left side of the screen you will find the element palette. Drag and drop the elements onto the diagram:
+2. The basic elements of BPMN processes are [tasks](/components/modeler/bpmn/tasks.md), or atomic units of work composed to create a meaningful result. Click on the start event and select the rectangular task element. This creates a task directly following the start event, connected by an arrow. Next, double-click the task and type in a name for the element. For example, `My Service Task`.
```

Comment:
`My Service Task` is a really bad name, as names shall be descriptive. 

I hence propose to remove the "For example My service task" line. You could instead add that names are meant to describe the activity being executed.